contents/blog/posthog-alternatives.md

```diff
@@ -0,0 +1,726 @@
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Fast to implement</td>
+    	</tr>
+		<tr>
+        	<td>Session replay</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Watch real user sessions</td>
+    	</tr>
+    	<tr>
+        	<td>Feature flags</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+          <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Manage feature access and rollouts</td>
+    	</tr>
+    	<tr>
+        	<td>A/B testing</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">CRO and test in production</td>
+    	</tr>
+    	<tr>
+        	<td>Group analytics</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Analyze groups (e.g. companies and teams)</td>
+    	</tr>
+    	<tr>
+        	<td>SQL access</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Write your own queries and filters</td>
+    	</tr>
+    	<tr>
+       	<td>Transparent pricing</td>
+      		<td className="text-center"><span className="text-red text-lg">✖</span></td>
+       	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Know exactly what you'll pay</td>
+    	</tr>
+    	<tr>
+       	<td>Billing limits</td>
+      		<td className="text-center"><span className="text-red text-lg">✖</span></td>
+       	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Prevent surprise bills</td>
+    	</tr>
+    	<tr>
+        	<td>Open source</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Open core, build your own integrations</td>
+   		</tr>
+    	<tr>
+        	<td>Cute hedgehogs</td>
+      		<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center">Joy</td>
+   		</tr>
+	</tbody>
```

Comment:
Nit: not sure I'd include this. Feels a bit off tone if the goal of this article is to inform. Up to you if you want to keep it!